lms/envs/common.py

```diff
@@ -304,7 +304,7 @@
     'ENABLE_COURSE_SORTING_BY_START_DATE': False,
 
     # Flag to enable new user account APIs.
-    'ENABLE_USER_REST_API': False,
+    'ENABLE_USER_REST_API': True,
```

Comment:
Since this is a feature branch, I am OK with turning this to True. Ultimately we will want to completely delete ENABLE_USER_REST_API.